unit1/exercises/e3/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"fmt"
+)
+
+type ErrNegativeSqrt float64
+
+func (e ErrNegativeSqrt) Error() string {
+	return fmt.Sprintln("cannot Sqrt negative number: -2")
```

Comment:
There must be no new line in such functions, only one line string. Also the actual number must be returned instead of hardcoded value.
fmt.Sprintf is the best option here

Reply:
Correct it on 1ecacf5634f553a429a6ca573ff7d7a2a2a79a28
As a part of this exercise, I found myself with the cast :)